src/QsCompiler/Tests.Compiler/AccessModifierTests.fs

```diff
@@ -19,55 +19,38 @@ type AccessModifierTests (output) =
                                                  [File.ReadAllLines("ReferenceTargets.txt").ElementAt(1)],
                            output)
 
-    member private this.Expect name (diagnostics : IEnumerable<DiagnosticItem>) = 
+    member private this.Expect name (diagnostics : IEnumerable<DiagnosticItem>) =
         let ns = "Microsoft.Quantum.Testing.AccessModifiers" |> NonNullable<_>.New
         let name = name |> NonNullable<_>.New
         this.Verify (QsQualifiedName.New (ns, name), diagnostics)
 
     [<Fact>]
     member this.``Callables`` () =
```

Comment:
It might be nice to add a suitable reference manually to the compilation unit (you would have to adapt CompilerTests.Compile to take references as argument), such that you can have a test or two where something should fail.

Reply:
Sorry, I'm not sure I understand - the tests where an inaccessible callable or type is used are in the "References" test. (Maybe I should get rid of the separate test and move them to the callables and types tests, though.)

Reply:
The tests that make sure internal callables and types can't be used in references have now been moved up to the callables and types tests, respectively.